db/migrate/20210211021215_add_player_to_match.rb

```diff
@@ -0,0 +1,5 @@
+class AddPlayerToMatch < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :matches, :player, foreign_key: true
```

Comment:
Is this going to be different since a match needs two players? I thought it might be different than what we've done in the past, but I may be over thinking it?

Reply:
Nah! I don't think you're overthinking it! How do we create a match and have it belong to multiple players? Or we could add an opponent(s) attribute to the player table and then just use a self-referential relationship since opponent is also a player?